test/parallel/test-debugger-repeat-last.js

```diff
@@ -10,7 +10,7 @@ const fixture = path.join(
 
 const args = [
   'debug',
-  `--port=${common.PORT}`,
+  '--port=0',
```

Comment:
>`--port=0` is something that I believe *should* work. 

See @bnoordhuis's https://github.com/nodejs/node/pull/5025 (and briefly @sam-github's https://github.com/nodejs/node/pull/12093).